hydrolib/core/io/dflowfm/bc/models.py

```diff
@@ -423,7 +281,186 @@ def __repr__(self) -> str:
         return str(representable)
 
 
-class TimeSeries(ForcingBase):
+class VectorForcingBase(ForcingBase):
```

Comment:
@tim-vd-aardweg: I just mentioned to consider renaming to `ScalarOrVectorForcingBase`, but forget about that, the name is fine.

Reply:
Okay :)